app/src/main/java/com/vinh/dictionary_1/screen/splash/SplashPresenter.java

```diff
@@ -41,7 +41,7 @@ public void onStop() {
     private void assetHandler() {
         SharedPrefsImplement sharedPrefs = SharedPrefsImplement.getInstance();
         if (sharedPrefs.get(Constant.PREF_DICT_DB_COPIED, Boolean.class)) {
-           //TODO move to home screen
+           mViewModel.switchToHomeActivity();
```

Comment:
ERROR: 'if' child have incorrect indentation level 11, expected level should be 12.